framework/platform/bpl/linux/bpl_cfg.cpp

```diff
@@ -335,5 +335,21 @@ int cfg_get_all_prplmesh_wifi_interfaces(BPL_WLAN_IFACE *interfaces, int *num_of
     return RETURN_OK;
 }
 
+bool cfg_get_persistent_db_enable(bool &enable)
```

Comment:
These are the only functions that return 'bool' on the file. Maybe it will be a good idea to not break the file convention.

Reply:
the original file convention is `c` based interface - and we no longer force the interface to be `c` and can now provide a `c++` style interface to any new APIs - refactoring of existing APIs might happen some day :-)